Cakefile

```diff
@@ -29,6 +29,12 @@ header = """
 # Used in folder names like `docs/v1`.
 majorVersion = parseInt CoffeeScript.VERSION.split('.')[0], 10
 
+# Patterns of names of test files which depend on currently absent features
+testFilesToSkip = [
+  'async.coffee'           unless try new Function 'async ()   => {}'
+  'async_iterators.coffee' unless try new Function 'async () * => {}'
+  'exponent_ops.coffee'    unless try new Function 'x = 3; x **= 2 ** 2; return x === 81'
+].filter _.identity
```

Comment:
Is the purpose of this is just to prevent `undefined` elements in the array? But such elements shouldn’t matter in `filename not in testFilesToSkip`. I know this is a bit nitpicky of an optimization, but I was scratching my head wondering what `filter _.identity` was doing 😄

Also please put a period at the end of your comment. The comments become the documentation in the annotated source, so they should be complete sentences.

Reply:
When the list was treated as a list of RegExps to be String::match'd against, undefined was unwelcome in the list. Now it's fine to leave them in. This is why we have code reviews. :)

I also didn't realize the Cakefile was subjected to the same comment -> documentation treatment as the files in src/. I'm adjusting my comments to account for this. Update forthcoming...